ui/apps/platform/src/Containers/Violations/violationTableColumnDescriptors.tsx

```diff
@@ -88,73 +89,75 @@ function EnforcementColumn({ original }: EnforcementTableCellProps): ReactElemen
     return <span>{message}</span>;
 }
 
-const tableColumnDescriptor = [
-    {
-        Header: 'Policy',
-        accessor: 'policy.name',
-        sortField: 'Policy',
-        Cell: ({ original }) => {
-            const url = `${violationsBasePath}/${original.id as string}`;
-            return (
-                <Tooltip content={original?.policy?.description || 'No description available'}>
-                    <Link to={url}>{original?.policy?.name}</Link>
-                </Tooltip>
-            );
+function getTableColumnDescriptors(filteredWorkflowView: FilteredWorkflowView) {
```

Comment:
Since the link to violation detail pages needs to include the current `filteredWorkflowView` in order to preserve navigation state in the breadcrumbs, we need to change this `const` to a function.

It is highly recommended to view this file diff with "hide whitespace" on to get the true scope of changes.